core/src/main/resources/static/css/habits.css

```diff
@@ -0,0 +1,16 @@
+input[type=range] {
+    -webkit-appearance: none;
+    background-color: #13AA57;
+    height: 2.5px;
+    border-radius: 2px;
+}
+
+input[type="range"]::-webkit-slider-thumb {
```

Comment:
Shouldn't we add properties with  -moz-* -ms-* prefixes  to ensure that it would be styled consistently across other browsers like Firefox, etc?